lib/DXIL/DxilUtilDbgInfoAndMisc.cpp

```diff
@@ -144,6 +144,17 @@ static Value *TryMegeWithNestedGEP(GEPOperator *GEP) {
       prevGEP = dyn_cast<GEPOperator>(AsCast->getPointerOperand());
   }
 
+  // If the addrspace cast changes the underlying type of the pointed to
```

Comment:
Nit: Would you mind changing the name of this function to TryMergeWithNestedGep ?